src/index.html

```diff
@@ -9,6 +9,10 @@
   <link rel="stylesheet" href="./styles/main.scss">
 </head>
 <body>
-  <h1>Stop watch</h1>
+  <div class="watch">
+    <div class="watch__minutes"></div>
+    <div class="watch__seconds"></div>
+    <div class="watch__center"></div>
+  </div>
 </body>
 </html>
```

Comment:
I know you change an existing file, but there are some lint problems :(
![image](https://user-images.githubusercontent.com/25587299/209169707-90dd5325-cb22-4915-91bd-c73346a84e2e.png)
Can you correct all of them, please?

Reply:
Yes, sure

Reply:
![image](https://user-images.githubusercontent.com/66367135/209206406-0a8e21ab-8637-42df-95e5-2f9cce76c6c6.png)

Reply:
You didn't correct these errors. 
Can you change your whole HTML with the correct indentations?
If you have any questions, you can ask in `fe_chat`

Reply:
This is the screenshot after the fix.